### DISCUSSION TITLE
What is the point of iconSet in quasar.config?

### DISCUSSION THREAD

**Discussion by BenJackGill** [quasarframework/quasar#16223] [General - Components / Directives / etc] [answered]
In the [docs](https://quasar.dev/options/quasar-icon-sets#introduction) it says we need to specify the icon set in `/quasar.config`.

For example:

```TypeScript
framework: {
  iconSet: 'svg-ionicons-v6'
}
```

I have done that without errors.

And then I have been using icons like this:
```Vue
<template>
  <q-btn dense color="purple" round :icon="ionHome" class="q-ml-md" />
</template>

<script setup lang="ts">
import { ionHome } from "@quasar/extras/ionicons-v6";
</script>
```

But if I comment out the `iconSet: 'svg-ionicons-v6'` line from `/quasar.config` the icon still works.

So why do we bother updating `/quasar.config` if the icons work without that?

**Comment by metalsadman** [comment, marked as answer]
https://quasar.dev/options/quasar-icon-sets#introduction `Quasar components have their own icons`. example notify icon displayed depending on `type` props, qselect's `default` dropdown icon, icon used when you use `clearable` on quasar form components, etc... what you are displaying is not an example usage of quasar iconSet.

**Reply by BenJackGill** [reply]
I see! Thanks for pointing that out :)
